source/includes/options-mongosqld.yaml

```diff
@@ -176,6 +176,36 @@ description: |
 optional: true
 ---
 program: mongosqld
+name: defaultAuthDatabase
+directive: option
+args: <database>
+description: |
+
+  Specifies the default source database for |bi| users. Set this value
```

Comment:
This phrasing is a little coy. I might replace the first two sentences with something like

```
Specifies the default authentication source database for connecting to the |bi|.
```